addons/sourcemod/scripting/ttt/ttt_shop.sp

```diff
@@ -1456,3 +1581,138 @@ int GiveClientItem(int client, char[] sItem)
 
     return false;
 }
+
+bool IsItemInInventory(int client, const char[] itemshort)
+{
+	StringMap InventoryMap = playerInventory[client][hInvItems];
+	if (InventoryMap == INVALID_HANDLE)
+	{
+		LogToFile(g_sLog, "Function: IsItemInInventory - Inventory not intialised for client: %d.", client);
+		return false;
+	}
+	
+	int amount = 0;
+	bool exists = InventoryMap.GetValue(itemshort, amount);
+	
+	if (exists && amount > 0)
+	{
+		return true;
+	}
+	
+	return false;
+}
+
+bool AddInventoryItem(int client, const char[] itemshort, bool stack)
```

Comment:
Maybe add an amount parameter that defaults to 1. So a plugin can give somebody multiple items at once.